### PR TITLE
docs: add info about settings --harmony-import-assertions flag

### PR DESCRIPTION
**Pull request by bartlomieju**

### CODE REVIEW

File: src/module.rs [hunk 1]
```diff
@@ -284,6 +284,8 @@ impl Module {
   /// Returns an empty Maybe<bool> if an exception occurred during
   /// instantiation. (In the case where the callback throws an exception, that
   /// exception is propagated.)
+  ///
+  /// NOTE: requires to set `--harmony-import-assertions` V8 flag.
```
Comment (ry) [inline src/module.rs:288]: Calling this at all need the flag?

Comment (bartlomieju) [reply src/module.rs:288]: Yes it does. I might have made a mistake removing old methods, could have left them with `deprecated_` prefix.

Comment (ry) [reply src/module.rs:288]: Ok.

Well - we apologize to any onlookers for the bumpy road.

**Review by ry — CHANGES_REQUESTED** [review]
LGTM

**Review by ry — APPROVED** [review]
LGTM

**Pull request merged**